cmd/echaindb/cmd/readChainconfig.go

```diff
@@ -47,7 +47,7 @@ Example:
 		}
 
 		log.Println("Opening database...")
-		db, err := rawdb.NewLevelDBDatabase(chainDBPath, 256, 16, "")
+		db, err := rawdb.NewLevelDBDatabase(chainDBPath, 256, 16, "", true)
```

Comment:
We should maybe think about getting rid of this executable since its functionality is being overtaken by that of `geth`'s `db` command (and never was developed beyond what I happened to need it for once upon a time).

Reply:
Proposed with https://github.com/etclabscore/core-geth/pull/366